src/components/RichLayout/Header.css

```diff
@@ -7,7 +7,7 @@
 }
 
 .spacer {
-  margin-bottom: var(--header-height);
+  padding-bottom: var(--header-height);
```

Comment:
On line 46 change `margin-bottom: var(--header-height-small);` to `padding-bottom: var(--header-height-small);`

Reply:
done